docs/source/configuration/authorization.mdx

```diff
@@ -98,199 +138,263 @@ So the `me` top level operation would not even be executed. When returning the r
 }
 ```
 
-If all the fields were removed, then the router would generate a query planner error indicating that the query is unauthorized.
+If every requested field requires authentication and a request is unauthenticated, the router generates a query planner error indicating that the query is unauthorized.
 
-## `@requiresScopes`
+### `@requiresScopes`
 
-The `@requiresScopes` directive restricts access to fields and types if the request did not present the right set of scopes, by loading from the request the context the object at key `apollo_authentication::JWT::claims`, and in that object the `scope` key, in the format defined by [OAuth2 access token scopes](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3), as a space separated list of scopes.
+The `@requiresScopes` directive marks fields and types as restricted based on the scopes you require.
+You can define the `@requiredScopes` directive on any subgraph schema like this:
 
-Depending on which scope set is presented by the request, different parts of the query may be available.
+```graphql
+directive @requiresScopes(scopes: [String!]!) on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
+```
+
+The directive should include a `scopes` argument that expects an array of required scopes.
+
+The directive validates the required scopes by loading the object at the `apollo_authentication::JWT::claims` key in a request's context.
+That object's `scope` key should contain a space separated list of scopes in the format defined by [OAuth2 access token scopes](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3).
+
+Depending on the scopes present on the request, the router filters out unauthorized fields and types.
+If a field's `scopes` array includes multiple scopes, the request must include _all_ required scopes to resolve it.
+
+#### Example `@requiresScopes` use case
+
+Imagine your social media platform has an ecommerce component.
+You need to be able to query for the following:
 
-Assuming we have this schema:
+- An authenticated user's profile (`me`)
+- Other user's emails
+- Products and the users who have ordered it
+
+Your schema may look something like this:
 
 ```graphql
 type Query {
-    me: User @requiresScopes(scopes: ["profile:read"])
-    products: [Products]
+  me: User @requiresScopes(scopes: ["profile:read"])
+  products: [Products]
 }
 
 type User {
-    id: ID
-    name: String
-    email: @requiresScopes(["user:read"])
+  id: ID
+  name: String
+  email: @requiresScopes(["user:read"])
 }
 
 type Product {
-    id: ID
-    name: String
-    amount: Int @requiresScopes(["employee", "inventory"])
-    orders: [User] @requiresScopes(["employee"])
+  id: ID
+  name: String
+  amount: Int @requiresScopes(["inventory:read"])
```

Comment:
@Geal , I standardized the format of the scopes to `object:action`, but let me know if you purposefully included two different formats for the scopes (`user:read` vs `employee`)

Reply:
I much prefer your format tbqh.